geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -54,10 +62,12 @@ object HBaseQueryPlan {
 case class EmptyPlan(filter: HBaseFilterStrategyType) extends HBaseQueryPlan {
   override val table: TableName = null
   override val ranges: Seq[Query] = Seq.empty
+  override val remoteFilters: Seq[HFilter] = Nil
   override val resultsToFeatures: Iterator[Result] => Iterator[SimpleFeature] = (i) => Iterator.empty
   override def scan(ds: HBaseDataStore): CloseableIterator[SimpleFeature] = CloseableIterator.empty
 }
 
+// JNH: Morally this is a 'local' / client-side ScanPlan
```

Comment:
Eh, let's re-write this comment as:

"This scan plan is used when remote code is not deployed.  As such 'resultsToFeatures' handles filtering and transformations."